emojis.json

```diff
@@ -480,7 +480,7 @@
     "category": "people"
   },
   "hankey": {
-    "keywords": ["poop", "shitface", "fail", "turd"],
+    "keywords": ["poop", "shitface", "fail", "turd", "shit", "poo"],
```

Comment:
Since `poo` is included `poop`, there's no need to have it as another keyword. Would you remove it? 👐